charts/cortex/templates/ingester/ingester-statefulset.yaml

```diff
@@ -224,7 +224,7 @@ spec:
                 values:
                 - {{ $availabilityCell }}
       tolerations:
-        {{- toYaml (index $.Values.ingester.availabilityCellsTolerations $availabilityCell) | nindent 8 }}
+      {{- toYaml (index $.Values.ingester.availabilityCellsTolerations $availabilityCell) | nindent 8 }}
```

Comment:
I guess with `nindent` you could just
```
tolerations: {{ toYaml (index $.Values.ingester.availabilityCellsTolerations $availabilityCell) | nindent 8 }}
```
as the purpose of `nindent` is to start with a linefeed (contrary to `indent`)